src/main/java/org/infoshareacademy/javaholics/route/RouteService.java

```diff
@@ -1,4 +1,149 @@
+                    System.out.println("Wybierz poprawny poziom trudności");
+                    levelOfDifficulty = scanner.nextInt();
+                }
+                if (levelOfDifficulty == 1) {
+                    System.out.println("Poziom trasy - Łatwy");
+                    route.setDifficulty(RouteDifficulty.EASY);
+                }
+                if (levelOfDifficulty == 2) {
+                    System.out.println("Poziom trasy - Średni");
+                    route.setDifficulty(RouteDifficulty.MEDIUM);
+                }
+                if (levelOfDifficulty == 3) {
+                    System.out.println("Poziom trasy - Wymagający");
+                    route.setDifficulty(RouteDifficulty.EXTREME);
+                }
+                if (levelOfDifficulty == 4) {
+                    System.out.println("Poziom trasy - Bardzo trudny");
+                    route.setDifficulty(RouteDifficulty.HARD);
+                }
+            } catch (InputMismatchException e) {
+                System.out.println("Wybierz poprawny poziom trudności trasy");
+                error = true;
+                scanner.nextLine();
+            }
+        } while (error);
+        System.out.println("Proponowany poziom trudności stworzonej trasy to " + route.getDifficulty());
+    }
+
+
+    public void routeType (){
+        System.out.println("Podaj typ trasy :  ");
+        System.out.println("1 - Trasa spacerowa ");
+        System.out.println("2 - Trasa rowerowa ");
+        System.out.println("3 - Trasa trekingowa ");
+        int typeRoute;
+        boolean error = false;
+        do {
+            error = false;
+            try {
+                typeRoute = scanner.nextInt();
+                while (typeRoute != 1 && typeRoute != 2 && typeRoute != 3) {
+                    System.out.println("Wybierz poprawny poziom trudności");
+                    typeRoute = scanner.nextInt();
+                }
+                if (typeRoute == 1) {
+                    System.out.println("Typ trasy - Trasa spacerowa");
+                    route.setType("Trasa spacerowa");
+                }
+                if (typeRoute == 2) {
+                    System.out.println("Typ trasy - Trasa rowerowa");
+                    route.setType("Trasa rowerowa");
+                }
+                if (typeRoute == 3) {
+                    System.out.println("Typ trasy - Trasa trekingowa");
+                    route.setType("Trasa trekingowa");
+                }
+            } catch (InputMismatchException e) {
+                System.out.println("Wybierz poprawny typ trasy");
+                error = true;
+                scanner.nextLine();
+            }
+        }while (error) ;
+        System.out.println(route.getType());
+    }
+
+    public void routeArea () {
+        System.out.println("W jakiej miejscowości chcesz zacząć? ");
+        route.setLocality(scanner.nextLine());
```

Comment:
przydałaby się weryfikacja, czy wpisano rzeczywiście nazwę - może sprawdzać, czy pierwszy i ostatni znak jest char A-z. Nie jest to krytyczne na te chwilę :)